tools/generator/templates/player/c/Makefile.jinja2

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+# Add all your sources (must be suffixed by *.cc) in here, separated by spaces
+CHAMPION_SOURCES = champion.c
+# Add all other files, like your headers in here, separated by spaces
+CHAMPION_HEADERS =
+
+# ----
+# Do not modify the lines below
+
+INTERFACE_SOURCES = interface.cc
+INTERFACE_HEADERS = api.hh
+
+CPPFLAGS = -MMD -MP
+CXXFLAGS = -ggdb3 -Wall -std=c++17 -fPIC
```

Comment:
We should add similar C flags.

In old generators the final command was `gcc -ggdb3 -Wall -Wextra -std=c11 -MD -MP -MF ./.prologin.o.d  -fPIC -c prologin.c -o prologin.o`